recurring-api-guide.md

```diff
@@ -787,9 +787,21 @@ Each specific charge on an agreement must be scheduled by the merchant.
 
 To create a charge use the [`POST:/recurring/v3/agreements/{agreementId}/charges`][create-charge-endpoint] endpoint.
 
-Also see check [Recommendations for `reference` and `orderId`](https://developer.vippsmobilepay.com/docs/knowledge-base/orderid) before creating charges.
-
-For agreements of type `variable`, also see [Recurring agreements with variable amount](#create-charge).
+For agreements of type `variable`, see [Recurring agreements with variable amount](#create-charge).
+
+>### `orderId` and `externalId`
+> The optional `orderId`-field will override the automatically generated `chargeId` for the charge if specified,
+> and will be used for all subsequent identification of the charge, and in settlement reports, unless an `externalId` is specified.
+> The `externalId`-field is a more flexible alternative to `orderId` which does not override the `chargeId`, but will be used in settlement reports.
+> 
+> This means that if both `orderId` and `externalId` are specified, 
+> the `orderId` will override `chargeId` and be used for all subsequent identification of the charge, 
+> while the `externalId` will be used in settlement reports.
+> 
+> As `externalId` does not have the same requirements for uniqueness as `orderId`, 
+> it is important that you understand how this will impact your settlements before using it.
+> 
+> Also see [Recommendations for `reference` and `orderId`](https://developer.vippsmobilepay.com/docs/knowledge-base/orderid)
```

Comment:
I intentionally added this section using > to put emphasis on it. please revise how to improve this if this is not a good way 😅